frontend/app/src/components/display/BalanceDisplay.vue

```diff
@@ -38,6 +40,31 @@ const usdValue = useValueOrDefault(
   Zero
 );
 
+const { currencySymbol } = storeToRefs(useGeneralSettingsStore());
+const { assetPrice } = useBalancePricesStore();
+
+const valueCurrency = computed(() => {
+  if (!get(calculateValue)) {
+    return 'USD';
+  }
+
+  return get(currencySymbol);
+});
+
+const valueInCurrency = computed(() => {
+  if (!get(calculateValue)) {
+    return get(usdValue);
+  }
+
+  const owned = get(amount);
+  const ethPrice = assetPrice('ETH');
+
+  if (isDefined(ethPrice)) {
+    return owned.multipliedBy(get(ethPrice));
```

Comment:
Why it hardcoded to ETH and not taken from prop `asset`?

Reply:
That is a good suggestion, thanks :)